dev/com.ibm.ws.jdbc/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -1351,6 +1351,20 @@
   <!-- By default, we disable the PostgreSQL PreparedStatement cache, in favor of the Liberty statement cache -->
   <AD id="preparedStatementCacheQueries" ibmui:group="Advanced" required="false" type="Integer" default="0" min="0"    name="%preparedStatementCacheQueries" description="%preparedStatementCacheQueries.desc"/>
   <AD id="readOnly"                      ibmui:group="Advanced" required="false" type="Boolean"                        name="%readOnly"            description="%readOnly.desc"/>
+  <AD id="ssl"                           ibmui:group="Advanced" required="false" type="Boolean"                        name="%ssl"                 description="%ssl.desc"/>
+  <AD id="sslCert"                       ibmui:group="Advanced" required="false" type="String"  ibm:type="location"    name="%sslCert"             description="%sslCert.desc"/>
+  <AD id="sslfactory"                    ibmui:group="Advanced" required="false" type="String"                         name="%sslfactory"          description="%sslfactory.desc"/>
```

Comment:
Still pending discussion on the default value here. We can either:
A) Leave it unset, thus making "PostgreSQL native SSL" the default way of configuring SSL
B) Default this to `org.postgresql.ssl.DefaultJavaSSLFactory`, thus making the default way of configuring SSL align with the rest of Liberty

Ultimately I am torn on which way is best. On the surface (B) seems better because it aligns well with the rest of the product, but (A) is safer because it reduces potential interference between the PostgreSQL driver and the user.